src/qa-test-eks.sh

```diff
@@ -146,4 +170,7 @@ if [ "$ACTION" = "destroy-shared" ]; then
     
     # Cleanup
     terragrunt destroy-all --terragrunt-working-dir "$WORKDIR" --terragrunt-source-update --terragrunt-non-interactive -input=false -auto-approve || true
```

Comment:
Should terragrunt really always succeed? What if it actually fails due to a timeout or a lock? Then there will be resources left behind?